### DISCUSSION TITLE
Quantized T5

### DISCUSSION THREAD

**Discussion by yamatazen** [vladmandic/sdnext#3282] [Q&A] [answered]
How to load the quantized T5 encoder in SD.Next?

**Comment by vladmandic** [comment]
simply select in *settings -> model -> text encoder* before or after model load  
you can also add it to quicksettings for fast access.

**Reply by yamatazen** [reply]
Does it download the encoder from Hugging Face?

**Reply by vladmandic** [reply, marked as answer]
yes.
if model already has t5, it will be used as-is so you don't need to download it from hf. but if you want to set explicit t5, then its downloaded.
